trezorlib/client.py

```diff
@@ -1082,6 +1083,21 @@ def self_test(self):
 
         return self.call(proto.SelfTest(payload=b'\x00\xFF\x55\xAA\x66\x99\x33\xCCABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!\x00\xFF\x55\xAA\x66\x99\x33\xCC'))
 
+    @field('address')
+    @expect(proto.RippleAddress)
+    def ripple_get_address(self, address_n, show_display=False):
+        return self.call(
+            proto.RippleGetAddress(
+                address_n=address_n, show_display=show_display))
+
+    @expect(proto.RippleSignedTx)
+    def ripple_sign_tx(self, n, transaction):
+        ripple.validate(transaction)
+        n = self._convert_prime(n)
```

Comment:
no more `_convert_prime` please

Reply:
yes, but not here. the `n` should already be the output of `parse_path`, that part is correct

Reply:
Sorry, I'm confused. So what should I use instead of `_convert_prime`?